RevenueCatUI/CustomerCenter/Actions/CustomerCenterConfigData.HelpPath+PurchaseInformation.swift

```diff
@@ -34,11 +34,21 @@ extension Array<CustomerCenterConfigData.HelpPath> {
             let isNonAppStorePurchase = purchaseInformation.store != .appStore
             let isAppStoreOnlyPath = $0.type.isAppStoreOnly
 
-            let isCancel = $0.type == .cancel
+            // skip AppStore only paths if the purchase is not from App Store
+            if isNonAppStorePurchase && isAppStoreOnlyPath {
+                return false
+            }
 
-            // if it's cancel, it cannot be a lifetime subscription
-            let isEligibleCancel = !purchaseInformation.isSubscription
-                || (!purchaseInformation.isCancelled &&  !purchaseInformation.isLifetimeSubscription)
+            if $0.type == .cancel {
+                // don't show cancel if there's no URL
+                if isNonAppStorePurchase && purchaseInformation.managementURL == nil {
+                     return false
+                }
+
+                return purchaseInformation.isSubscription
+                    && !purchaseInformation.isLifetimeSubscription
```

Comment:
There's no way something `isSubscription == true` and `isLifetimeSubscription == true` right? so I think just checking `isSubscription` is enough? Or do promotionals break that?